html-forms/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="https://example.com/register" method="post">
+      <label for="email">Email</label>
+      <br />
+      <input type="email" id="email" name="email" required />
+      <br /><br />
+      <label for="coding">Why is coding cool</label>
+      <br />
+      <select id="coding" name="coding">
```

Comment:
This is fine for this exercise, but in the future I would get out of the habit of using `<br />` elements for new lines. Generally speaking the building block of HTML is going to be your `<div>` element which is a **block** element. This will start on a new line and you can use this for new line layout and structure.